homeassistant/components/yamaha/media_player.py

```diff
@@ -215,8 +215,10 @@ def update(self) -> None:
         """Get the latest details from the device."""
         try:
             self._play_status = self.receiver.play_status()
+            self._attr_available = True
```

Comment:
Please move this line down below the `return`. We only want to wrap the line that can raise in the try.. except block.